NiL.JS/BaseLibrary/Array.cs

```diff
@@ -63,6 +63,15 @@ public Array(double length)
         }
 
         [DoNotEnumerate]
+        internal Array(JSValue[] data)
```

Comment:
It causes errors for you with `public`? I have never seen those but I wanted it to be `internal` anyway.
Yes, `[DoNotEnumerate]` is useless that way.